consensus/fork.go

```diff
@@ -98,6 +98,7 @@ func (s *State) generateAndApplyDiff(bn *blockNode) (err error) {
 	s.currentBlockID = bn.block.ID()
 	s.currentPath[bn.height] = bn.block.ID()
 
+	// TODO: minerSubsidy isn't used anywhere?
```

Comment:
actually, b/c of all the currency changes...

you can delete the `minerSubsidy :=` declaration. and the ` for _, fee` loop.